extensions/wikia/DesignSystem/i18n/en.json

```diff
@@ -55,7 +55,7 @@
   "global-navigation-notifications-title": "Notifications",
   "global-navigation-search-cancel": "Cancel",
   "global-navigation-search-placeholder-fandom": "Search Stories and Wikis...",
-  "global-navigation-search-placeholder-in-wiki": "Search __sitename__",
+  "global-navigation-search-placeholder-in-wiki": "Search $1",
```

Comment:
Can we re-download translations?
This string had been changed recently: https://github.com/Wikia/design-system-i18n/blob/master/i18n/en/design-system.json#L59